Jenkinsfile

```diff
@@ -175,10 +176,9 @@ def job = {
                                 set +x
 
                                 LOGIN_CMD=$(aws ecr get-login --no-include-email --region us-west-2)
-
                                 $LOGIN_CMD
-                            '''
-                            sh '''
+
+                                set -x
                                 echo $ARTIFACTORY_PASSWORD | docker login confluent-docker.jfrog.io -u $ARTIFACTORY_USERNAME --password-stdin
                             '''
                             writeFile file:'create-pip-conf-with-jfrog.sh', text:libraryResource('scripts/create-pip-conf-with-jfrog.sh')
```

Comment:
you can move these up to line 171 along with other call to writeFile, and then you only need one sh block.

Reply:
Done.